exec/java-exec/src/main/java/org/apache/drill/exec/physical/base/AbstractGroupScanWithMetadata.java

```diff
@@ -449,10 +454,22 @@ public boolean supportsLimitPushdown() {
     return true;
   }
 
+  public void setLimit (int limitToPushDown) {
```

Comment:
This method is not used anymore. Could you please remove it?

Reply:
@vvysotskyi 
Removed extra method.